docs/assets/css/tabs.less

```diff
@@ -1,166 +1,103 @@
-/* ==========================================================================
-   tabs
-   ========================================================================== */
-// Based on gov.uk Tabs: https://design-system.service.gov.uk/components/tabs/
-// breakpoints:
-// 40.0525em
-// 40.0625em; 641px
-// if we change them in this CSS we also have to fork and release our own JS with updated breakpoints
-
-
-@media (min-width: 40.0625em) {
-    .govuk-tabs {
-        margin: 0 -1rem;
-    }
-}
-
-.govuk-tabs__list {
-    border-top: 1px solid @gray;
+.m-tabs {
+    width: 20em;
+    font-family: "lucida grande", sans-serif;
```

Comment:
Probably don't want to use this font-family. Where's this coming from? Is that the govuk default? I'd just leave `font-family` out altogether unless it looks disasterous.